<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База детского центра</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="butAddClient">
         <property name="text">
          <string>Добавить семью</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butEditClient">
         <property name="text">
          <string>Коррекция</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butDelClient">
         <property name="text">
          <string>Исключение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butAddChild">
         <property name="text">
          <string>Добавить ребенка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butEditChild">
         <property name="text">
          <string>Коррекция</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butDelChild">
         <property name="text">
          <string>Исключение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butSetCats">
         <property name="text">
          <string>Установить категории</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButList">
         <property name="text">
          <string>Создать список</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butDiag">
         <property name="text">
          <string>Создать диаграмму</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButFind">
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="butRefs">
         <property name="text">
          <string>Справочники</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTableView" name="tableClients"/>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Дети</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableChildrens">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Информация</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textInfo">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labInfo">
         <property name="text">
          <string>Состояние</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
